src/main/java/com/skynest/uitesting/models/User.java

```diff
@@ -0,0 +1,28 @@
+package com.skynest.uitesting.models;
+
+import com.github.javafaker.Faker;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import static com.skynest.uitesting.utils.RandomGenerator.generateValidEmail;
+
+@Getter
+@RequiredArgsConstructor
+public class User {
+    private static final Faker faker = new Faker();
+    private final String firstName;
+    private final String lastName;
+    private final String emailAddress;
+    private final String phoneNumber;
+    private final String homeAddress;
+    private final String password;
+    private final String confirmPassword;
+
+    public static User generateValidAccount() {
```

Comment:
Might be more consistent to call this method generateValidUser since it returns a User instance and account doesn't appear anywhere else. Alternatively, you could rename this class to Account.